snippets/pyaloha/protocol.py

```diff
@@ -145,6 +145,8 @@ class CustomEncoder(json.JSONEncoder):
     def default(self, obj):
         if hasattr(obj, '__dumpdict__'):
             return obj.__dumpdict__()
+        if isinstance(obj, bytes):
+            return obj.decode()
```

Comment:
encode?

Reply:
no. bytes can be decoded. strings can be encoded with encoding into bytes.